tox.ini

```diff
@@ -24,6 +24,8 @@ envlist =
 
     {pypy,py2.7,py3.5,py3.6,py3.7,py3.8}-bottle-0.12
 
+    {pypy,py2.7,py3.5,py3.6,py3.7,py3.8}-falcon-1.4
```

Comment:
I overlooked this.. you need to write `falcon-{1.4,dev}` to actually test the dev version. Bottle is broken here as well.

Reply:
I've just changed it to 2.0 actually. 2.0 is in its third release candidate so testing 3.0 dev feels a bit early, right?